### PR TITLE
Mobile v2.2 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Review by m1lt0n — APPROVED** [review]

File: source/deploy/mobile-app-changelog.md [hunk 1]
```diff
@@ -2,12 +2,51 @@
...
+
+### Compatibility
+ - **Upgrade to server version v7.1.0 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/upgrade/extended-support-release.html) (ESR) 6.3.0 has ended and upgrading to server ESR v7.1.0 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
+ - iPhone 5s devices and later with iOS 12.1+ is required.
+
+### Improvements
+ - Added the ability to set the mobile notifications preferences per channel.
+ - Calls: Added slash commands to start/stop call recordings (``/call recording [start|stop]``).
+ - Calls: Implemented a glare free negotiation.
```
Comment (cwarnermm) [inline source/deploy/mobile-app-changelog.md:24]: Curious to know more about what this means

Comment (cpoile) [inline source/deploy/mobile-app-changelog.md:24]:
```suggestion
 - Calls: Implemented glare free negotiation.
```

File: source/deploy/mobile-app-changelog.md [hunk 2]
```diff
@@ -2,12 +2,51 @@
...
+ - The Help link now is not converted to lowercase.
+ - Added minor performance improvements on sending a message.
+ - Archived channels show a more detailed warning.
+
+### Bug Fixes
+ - Calls: Fixed a rare case where the Join Call banner showed that a call started "53 years ago".
+ - Calls: fixed a crash on joining calls.
```
Comment (cpoile) [inline source/deploy/mobile-app-changelog.md:31]:
```suggestion
 - Calls: Fixed a crash on joining calls.
```

File: source/deploy/mobile-app-changelog.md (hunk 1 above)
Comment (cpoile) [reply source/deploy/mobile-app-changelog.md:24]: It's a problem with Webrtc where if two people try to simultaneously connect to each other, the negotiation fails (they can't fully connect).

**Review by cpoile — APPROVED** [review]
A couple of wording suggestions. Thanks!

File: source/deploy/mobile-app-changelog.md (hunk 1 above)
Comment (cwarnermm) [reply source/deploy/mobile-app-changelog.md]: Thanks, @cpoile! Do we expect end users to be familiar with the term "glare" in this context? If not, we may want to use language more familiar to the reader.

**Review by streamer45 — APPROVED** [review]

File: source/deploy/mobile-app-changelog.md (hunk 1 above)
Comment (cpoile) [reply source/deploy/mobile-app-changelog.md]: Maybe: 
```suggestion
 - Calls: Implemented glare free negotiation. This fix prevents potential negotiation problems when two clients try to connect simultaneously.
```

**Review by cpoile — APPROVED** [review]
Thanks Amy, Carrie

File: source/deploy/mobile-app-changelog.md [hunk 3]
```diff
@@ -2,12 +2,52 @@
...
+- Release Date: March 16, 2023
+- Server Versions Supported: Server v7.1.0+ is required. Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device.
+
+### Compatibility
+ - **Upgrade to server version v7.1.0 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/upgrade/extended-support-release.html) (ESR) 6.3.0 has ended and upgrading to server ESR v7.1.0 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
+ - iPhone 5s devices and later with iOS 12.1+ is required.
+
+### Improvements
+ - Added the ability to set the mobile notifications preferences per channel.
```
Comment (cwarnermm) [inline source/deploy/mobile-app-changelog.md:22]:
```suggestion
 - Added the ability to set mobile notifications preferences per channel.
```

File: source/deploy/mobile-app-changelog.md [hunk 4]
```diff
@@ -2,12 +2,52 @@
...
+ - Calls: Added slash commands to start/stop call recordings (``/call recording [start|stop]``).
+ - Calls: Implemented glare free negotiation. This fix prevents potential negotiation problems when two clients try to connect simultaneously.
+ - The Help link now is not converted to lowercase.
+ - Added minor performance improvements on sending a message.
+ - Archived channels show a more detailed warning.
+
+### Bug Fixes
+ - Calls: Fixed a rare case where the Join Call banner showed that a call started "53 years ago".
+ - Calls: Fixed a crash on joining calls.
+ - Fixed an issue where tapping "Send Message" in a user’s profile pop-over did not open a Direct Message channel.
```
Comment (cwarnermm) [inline source/deploy/mobile-app-changelog.md:33]:
```suggestion
 - Fixed an issue where tapping **Send Message** in a user’s profile pop-over did not open a Direct Message channel.
```

File: source/deploy/mobile-app-changelog.md [hunk 5]
```diff
@@ -2,12 +2,52 @@
...
+ - Fixed an issue where an incorrect skin tone was applied to emojis selected in the emoji picker.
+ - Fixed an issue where the channel list displayed Direct Messages with users that had been deactivated.
```
Comment (cwarnermm) [inline source/deploy/mobile-app-changelog.md:35]:
```suggestion
 - Fixed an issue where the channel list displayed Direct Messages with user accounts that had been deactivated.
```

File: source/deploy/mobile-app-changelog.md [hunk 6]
```diff
@@ -2,12 +2,52 @@
...
+ - Fixed an issue with searching for channels and users with non-Latin characters.
```
Comment (cwarnermm) [inline source/deploy/mobile-app-changelog.md:36]:
```suggestion
 - Fixed an issue with searching for channels and users that contain non-Latin characters.
```

File: source/deploy/mobile-app-changelog.md [hunk 7]
```diff
@@ -2,12 +2,52 @@
...
+ - Fixed an issue where selecting an item from the autocomplete doubled tilde and slash characters.
+ - Fixed an "Unable to reset your password" issue.
+ - Fixed an issue where the Group Message member count showed as 0 on GraphQL enabled instances.
+ - Fixed an issue with missing posts in a thread when a post got deleted.
```
Comment (cwarnermm) [inline source/deploy/mobile-app-changelog.md:40]:
```suggestion
 - Fixed an issue with missing posts in a thread when a post gets deleted.
```

**Review by cwarnermm — COMMENTED** [review]
Provided feedback inline.

**Pull request merged**